Feature/game overview

thuanvmse161224 opened the pull request.

thuanvmse161224 committed:
feat: swagger

thuanvmse161224 committed:
chore: new api link, linting

dg-panh commented on src/Layout/GameDashboardLayout/Navbar/Navbar.tsx:
```diff
@@ -58,9 +61,47 @@ const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop) => prop !== 'open'
+	maxWidth: '100%', // Ensure the username doesn't exceed the container's width
+}))
+
+
+type GameResponse = {
+	isError: boolean,
+	message?: string,
+	result?: Game,
+	responseException?: {
+		exceptionMessage: string,
+	},
+};
+
```
cái này là failure response chung ấy -> khai báo vào type file đi

thuanvmse161224 replied:
rồi h em mới thấy nè chị

dg-panh commented on src/pages/GameOverview/GameOverview.tsx:
```diff
@@ -0,0 +1,193 @@
+import DialogAPI from './components/DialogAPI'
+import UpdateGameDialog from './components/UpdateGameDialog'
+
+
+type GameResponse = {
+	isError: boolean,
+	message?: string,
+	result?: Game,
+	responseException?: {
+		exceptionMessage: string,
+	},
+};
+
```
nè dùng nhiều nè nên khai báo để dùng chung

thuanvmse161224 replied:
done, check my newest commit

thuanvmse161224 committed:
chore: misc fixes, read details below
+ change landing color
+ remove requiredMsg createGame (optional now)
+ change url regex
+ remove remaining DevTool react-hook-form
+ adding go back My Project button

thuanvmse161224 committed:
chore: remove required in gameupdateform

thuanvmse161224 committed:
refactor: change Game fetching procedure
+ change HttpResponse to generic
+ add isError to FailureResponse
+ adding postfix GMS to both http and failure response

dg-panh committed:
style: landing page color

dg-panh committed:
chore: item of navbar

thuanvmse161224 committed:
refactor: change swagger.json to describe new server url

thuanvmse161224 committed:
Merge branch 'feature/gameOverview' of https://github.com/FPTU-GHub-Capstone/ghub-app into feature/gameOverview
